qiskit/pulse/schedule.py

```diff
@@ -150,6 +150,33 @@ def __init__(self,
                 time, sched = 0, sched_pair
             self._mutable_insert(time, sched)
 
+    @classmethod
+    def inherit_from(cls, other_program: Any, name: Optional[str] = None) -> 'Schedule':
```

Comment:
I think I would prefer `initialize_from` as `inherit` implies a persistent coupling to the parent `Schedule`.